cobalt/h5vcc/h5vcc_net_log.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "cobalt/h5vcc/h5vcc_net_log.h"
+
+#include "base/files/file_util.h"
+#include "base/path_service.h"
+#include "cobalt/base/cobalt_paths.h"
+
+namespace cobalt {
+namespace h5vcc {
+
+namespace {
+const char* kOutputNetLogFilename = "h5vcc_netlog.json";
+
+base::FilePath filepath_to_absolute(
+    const base::FilePath& output_path_relative_to_logs) {
+  base::FilePath result;
+  base::PathService::Get(cobalt::paths::DIR_COBALT_DEBUG_OUT, &result);
+  return result.Append(output_path_relative_to_logs);
```

Comment:
Let's make sure this can't be abused to read any file off the filesystem.

Reply:
cools, yeah, no need to make this configurable, I've removed that option to pass a filename.